src/vs/base/test/browser/ui/splitview/splitview.test.ts

```diff
@@ -73,12 +73,12 @@ suite('Splitview', () => {
 	});
 
 	teardown(() => {
-		container = null;
+		container = null!;
```

Comment:
Actually the right thing to do here is to just remove the whole `teardown` section and simply never set `container` to `null` and let the garbage collector do its thing.